datastore/emulator.go

```diff
@@ -108,7 +108,12 @@ func Emulator(ctx context.Context) <-chan struct{} {
 	}
 
 	// This is emulating the old `aetest` framework that has a hardcoded "testapp" value for project
-	if err := os.Setenv("GOOGLE_CLOUD_PROJECT", fmt.Sprintf(fakeProjectID)); err != nil {
+	if err := os.Setenv("GOOGLE_CLOUD_PROJECT", fakeProjectID); err != nil {
+		log.Errorf("couldn't set env GOOGLE_CLOUD_PROJECT: %v", err)
+	}
+
+	// This env is set to pretend we're running on AppEngine
+	if err := os.Setenv("GAE_ENV", "standard"); err != nil {
```

Comment:
Check is for GAE_ENV, error is for GOOGLE_CLOUD_PROJECT

Reply:
Shame on me T__T Fixed.